scripts/openmc-track-to-vtk

```diff
@@ -65,7 +65,7 @@ def main():
         # Write coordinate values to points array.
         if fname.endswith('.binary'):
             track = open(fname, 'rb').read()
-            coords = [struct.unpack("ddd", track[24*i : 24*(i+1)])
+            coords = [struct.unpack("ddd", track[24*i:24*(i+1)])
```

Comment:
I personally think this looks too cluttered without the whitespace, and the whitespace is allowed in PEP 8 (see the third bullet in https://www.python.org/dev/peps/pep-0008/#pet-peeves).